### ISSUE TITLE
Adding Tair or VPDleaf to the required survey variable

### ISSUE THREAD

**Issue by JulienLamour** [ess-dive-community/essdive-leaf-gas-exchange#12]
* Documentation reference (URL): https://github.com/ess-dive-community/essdive-leaf-gas-exchange/blob/main/docs/requiredVariables.md
* Submitter: 
* Justification (why is this change necessary?):
The aim of survey gas exchange measurements is often to calibrate stomatal conductance models. See for example: Lamour et al. 2023 or Wu et al. 2020. VPDleaf is needed in many stomatal conductance models, including Medlyn et al. 2011 which is used in most vegetation models. With the current required variables, it is not possible to estimate the parameters of the Medlyn model. 
*I suggest the following changes: I suggest to add VPDleaf to the required variables for the survey measurements. Or Tair so VPDleaf can be calculated using the other required variables.

Lamour, J., Souza, D.C., Gimenez, B.O., Higuchi, N., Chave, J., Chambers, J. et al. (2023) Wood-density has no effect on stomatal control of leaf-level water use efficiency in an Amazonian forest. Plant, Cell & Environment, 46, 3806–3821.

Wu J, Serbin SP, Ely KS, et al. The response of stomatal conductance to seasonal drought in tropical forests. Glob Change Biol. 2020; 26: 823–839.

**Comment by regnans** [comment]
Hi Julien, 
Thanks for your suggestion. Do you think for simplicity it is worth adding both VPDleaf and Tair as required variables? Or if it were only one, which would be more useful? 

We are in the process of updating this reporting format, including restructuring of the layout on GitHub to improve accessibility, and some alignment with ESS-DIVE standards. This change to required variables for Survey measurements can be made as part of that update. 

- Kim

**Comment by regnans** [comment]
Although, I just want to check that Tleaf is not enough to calculate VPDleaf. On the 6400 VPDleaf is calculated with Tleaf, not Tair. If that is the case, then the required variables of Tleaf, RH and Patm allow calculated of VPDleaf.

**Comment by JulienLamour** [comment]
Hi Kim, 

The leaf to air vapour pressure deficit needs Tair if it is calculated using RH, otherwise you need an estimate of the water pressure at the leaf surface. 

I think Tair doesn't appear directly in the Licor calculation because they use the air vapor pressure using Patm and H2Os. Since H2Os is not a required variable, I think we always miss one variable to be able to calculate VPDleaf.

You can follow the calculations made with the licor but I dont think you ll be able to calculate VPDleaf with the "required variables". Tell me if I am wrong.

Cheers

Julien

**Comment by regnans** [comment]
Ah, yes, you are correct, H2OS is used in the LI-COR calculation. I was focused on the T part!  

In thinking about which variable would be best to include, I see they have different "pros". Tair is measured directly by the instruments, rather than calculated, and may be used for other things. However VPDleaf can be used directly for the parameter estimates. Do you have any preference for one, or both? Thanks again for your input on this.

**Comment by JulienLamour** [comment]
I agree with the pros and cons and I don't have a solid preference. For my needs, VPDleaf is more useful ;p

**Comment by regnans** [comment]
I will add both as required variables for survey. It's essentially no additional burden, and may allow more flexible uses in the future. I will keep this issue open until this is executed (I need to have some broader conversation with ESS-DIVE before releasing V2 as it has been on hold due to staff changes).
